screens/HomeFeed/Tabs/DMFeed/index.tsx

```diff
@@ -174,6 +174,9 @@ const DMFeed = ({navigation}: Props) => {
       />
       {showDM ? (
         <Pressable
+          onPress={() => {
+            navigation.navigate('DmAllMembers');
```

Comment:
can we use any constant files to maintain screen name like `DmAllMembers`. Also use `DM` instead of `Dm`, to main sync?